borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
Yes, i expect KeyManager to expose more commands in the future. I did’t look much at borg’s cmdparsing yet. Would we want a group still being dispatched in archiver or can/want we delegate everything into the key manager? Archiver is quite bit and unfocused.